shell/platform/embedder/embedder.cc

```diff
@@ -125,6 +125,14 @@ FlutterResult FlutterEngineRun(size_t version,
     };
   }
 
+  std::function<bool()> make_resource_current_callback = nullptr;
+  if (config->open_gl.make_resource_current != nullptr) {
+    make_resource_current_callback =
+        [ ptr = config->open_gl.make_resource_current, user_data ]()->intptr_t {
```

Comment:
The `-> intptr_t` will introduce an unnecessary cast. You could just skip it.

Reply:
Done